apps/include/opt.h

```diff
@@ -271,17 +271,20 @@
 # define OPT_PROV_ENUM \
         OPT_PROV__FIRST=1600, \
         OPT_PROV_PROVIDER, OPT_PROV_PROVIDER_PATH, \
+        OPT_PROV_PROVIDER_CONFIG_FILE, \
         OPT_PROV__LAST
 
 # define OPT_PROV_OPTIONS \
         OPT_SECTION("Provider"), \
         { "provider_path", OPT_PROV_PROVIDER_PATH, 's', "Provider load path (must be before 'provider' argument if required)" }, \
-        { "provider", OPT_PROV_PROVIDER, 's', "Provider to load (can be specified multiple times)" }
+        { "provider", OPT_PROV_PROVIDER, 's', "Provider to load (can be specified multiple times)" }, \
+        { "provider_config", OPT_PROV_PROVIDER_CONFIG_FILE, 's', "Provider config file to load (can be specified multiple times)." }
```

Comment:
The parenthesis should be dropped, that's not how you have implemented it.

Reply:
Removed this file also..  Multiple configs could be loaded - so the parenthesis are valid.